Src/VimWpf/Implementation/CommandMargin/CommandMarginController.cs

```diff
@@ -847,7 +847,7 @@ private void UpdateForPasteWait(WpfTextChangedEventArgs e)
                     // Now we need to update the command line.  During edits the controller is responsible
                     // for manually updating the command line state.  Also we have to keep the caret postion
                     // correct
-                    var name = RegisterName.OfChar('c');
+                    var name = RegisterName.OfChar(c);

```

Comment:
I checked just to be sure and yep this is my fault 😦

Reply:
Ha, ha, I bet you were manually testing with the `c` register!